requirements/base.txt

```diff
@@ -1 +0,0 @@
-fire
```

Comment:
in such case we do not need the file at all as we say base is dependency free :)

Reply:
You're right! Let me adjust for that now.

Reply:
Addressed in 4003a68.